dallinger/recruiters.py

```diff
@@ -727,6 +736,60 @@ def get_qualifications(self, hit_id, sandbox):
             "peripheral_requirements": details["peripheral_requirements"],
         }
 
+    def screen_out(self, participant: Participant):
+        """Screen-out a participant.
+
+        Parameters:
+            participant: Participant
+                The participant to screen-out.
+        """
+        if self.screen_out_allowed(participant):
+            try:
+                return self.prolificservice.screen_out(
+                    study_id=self.current_study_id,
+                    submission_id=participant.assignment_id,
+                    bonus_per_submission=participant.base_pay + participant.bonus,
+                    increase_places=self.config.get("auto_recruit"),
```

Comment:
We need to proper handle the case where Prolific rejects the screen out. We could throw an error like `ProlificScreenOutDenied`?

Reply:
Added logic to raise a `ProlificScreenOutDenied` exception.